app/javascript/react/components/PizzaForm.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, } from "react";
+
+const PizzaForm = (props) => {
+  const [newPizza, setNewPizza] = useState({
+    product_name: "",
+    cost: "",
+    brand_id: null
+  })
+
+  const handleChange = (event) => {
+    event.preventDefault()
+    setNewPizza({
+      ... newPizza,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  } 
+
+  const formSubmitCallback = (event) => {
+    event.preventDefault()
+
+    const tempPizza = newPizza
+    tempPizza.brand_id = props.brand_id
```

Comment:
instead of passing a `brand_id` around as props, I would recommend your POST pizza fetch request be made to `/api/v1/brands/:brandId/pizzas`, that way the id is only needed to make the POST fetch request